.github/workflows/build-ci.yml

```diff
@@ -19,15 +19,24 @@ jobs:
         with:
           java-version: '8.0.292'
 
+      - name: Get Semver
+        id: semver
+        if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
```

Comment:
I don't know if this is possible, but `startsWith(github.ref, 'refs/heads/release/v')` is used in a bunch of places, can that be initialized at the start and re-used? Maybe as an env variable?

Reply:
There's a lot of workflow yml duplication (ie, the java version too) - I don't think there's a "proper" way to do de-duplication at the configuration level (GH yaml doesn't support yaml anchors :/). Changing the environment *may* work, but is technically fragile since the values can change at runtime. Will investigate.